prometheus-ksonnet/lib/config.libsonnet

```diff
@@ -99,14 +99,24 @@
       for c in std.objectFields($._config.alertmanager_clusters)
     },
 
-    // Shortcut to alertmanager_clusters entry for this cluster.
+    // Backwards compatible base entry $.alertmanager_cluster_self
```

Comment:
this comment no longer makes sense here?

Reply:
how does it not make sense?